packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -647,13 +647,14 @@ export function useRemainingTime(tx: MergedTransaction) {
 
     if (isTransferConfirmed(tx)) {
       setIsConfirmed(true)
+      setRemainingMinutes(0)
     } else {
-      setRemainingTime(canBeClaimedDate.fromNow(true).toString())
+      setRemainingMinutes(canBeClaimedDate.diff(dayjs(), 'minutes'))
     }
   }, 2000)
 
   return {
-    remainingTime,
+    remainingMinutes,
```

Comment:
also rename `remainingMinutes` to `remainingMinutesCctp` here so you don't have to rename the var when using the hook